src/action.c

```diff
@@ -121,7 +121,8 @@ enum action_type {
 	ACTION_TYPE_TOGGLE_TABLET_MOUSE_EMULATION,
 	ACTION_TYPE_TOGGLE_MAGNIFY,
 	ACTION_TYPE_ZOOM_IN,
-	ACTION_TYPE_ZOOM_OUT
+	ACTION_TYPE_ZOOM_OUT,
+	ACTION_TYPE_WARP_CURSOR
```

Comment:
Not specific to this PR, but I suggest adding a comma also to the last element in the enum so that we can minimize the diff when we add more actions. AFAIK we don't have any coding rule to prohibit that.